devel/rules.yml

```diff
@@ -0,0 +1,57 @@
+subject: namespace:ns
```

Comment:
used https://github.com/istio/istio/pull/337/files#diff-ad310135e477fb2632b2043cc24c646fR4
and it same as the one in mixer/testdata minus the attributes